R/zzz.R

```diff
@@ -73,11 +64,13 @@
 .objectDefinitionHelper <- function(object) {
   if (methods::is(object, "CsvFileTable")) {
     # reading from csv
-    unlockBinding("asDataFrame", object)
```

Comment:
This one may merit further investigation. I don't believe the lock and unlock binding calls are related to PythonEmbedInR. All I can say with certainty is that removing them did fix errors and did not appear to adversely affect `asDataFrame` functionality.

Reply:
Can you speak to the errors?  It would be preferred if these were commented out with a comment about this so that if our users do run into issues without this, we can look into it.

Reply:
On the very next line I added the comment:
# Removed due to Error in unlockBinding("asDataFrame", object) : no binding for "asDataFrame"
Let me know if this is sufficient